README.md

```diff
@@ -183,35 +183,36 @@ This is the most tested way to launch.
     ```bash
     mkdir data
     ```
-4) Launch the system (Option 1):
-    * On Mac:
+4) Clone the repository
     ```bash
     git clone git@github.com:deephaven-examples/deephaven-ib.git
-    cd deephaven-ib/docker/dev/build.sh
+    cd deephaven-ib/docker/dev/ && ./build.sh
+    ```
```

Comment:
1) This step is both a clone and a build.
2) This step is only needed when launching on the mac.  That's why it was listed there.  Currently, on mac, you need to build the images, since we are not publishing official images.